apps/mobile/.env.exmple

```
@@ -0,0 +1 @@
+BASE_URL="http://api.github.com"
```

```suggestion
GITHUB_BASE_URL="http://api.github.com"
```

Coisa até boba, mas será que não compensa já colocar bonito já?

---

não vamos usar a URL do github, isso é somente um exemplo de URL que coloquei. Com certeza depois essa url será substituída pela URL da api do backend, esse foi o intuito, como ainda não temos URL definida, nem qualquer outro endpoint